tools/trusted-app-client/src/test/java/edu/emory/cci/bindaas/trusted_app_client/test/TestTrustedAppClient.java

```diff
@@ -71,15 +75,16 @@ public void testGetShortLivedAPIKey()
 		String applicationID = "demo-id";
 		String applicationSecret = "demo-secret-key";
 		String randomUser = UUID.randomUUID().toString();
+		String protocol = "api_key";
 		Long epochTimeExpires = (new Date()).getTime() + 1000*60; // expires in 1 min
 		String comments = "junit testing";
 		TrustedAppClientImpl client = new TrustedAppClientImpl(url,applicationID, applicationSecret);
 		try {
-			APIKey masterKey = client.authorizeNewUser(randomUser, epochTimeExpires, comments);
-			Assert.assertNotNull(masterKey);
-			System.out.println(masterKey);
-			APIKey shortKey = client.getShortLivedAPIKey(randomUser, 10);
-			System.out.println(shortKey);
+			JsonObject masterResponse = client.authorizeNewUser(protocol, randomUser, epochTimeExpires, comments);
+			Assert.assertNotNull(masterResponse);
+			System.out.println(masterResponse);
+			JsonObject shortResponse = client.getShortLivedAuthenticationToken(protocol, randomUser, 10);
+			System.out.println(shortResponse);
 			
 		} catch (Exception e) {
 			e.printStackTrace();
```

Comment:
replace it by log.error() and wherever applicable.

If you find e.printStackTrace() anywhere when you work on the code, please replace them with log.error() as well.